app/read_models/received_on_reports/configuration.rb

```diff
@@ -5,7 +5,7 @@ class Configuration
     ].freeze
 
     CLOSING_EVENTS = [
-      Reviewing::SendBack, Reviewing::Completed
+      Reviewing::SentBack, Reviewing::Completed
```

Comment:
😮

Reply:
I know 😞

Reply:
Easily done. And we all missed it in peer review too!